src/routes/api/listed_apps.js

```diff
@@ -1,9 +1,31 @@
 module.exports = function (express) {
+  /**
+   * @var {module} router
+   * This sets the express.Router function to the variable, router.
+  */
   var router = express.Router();
+
+  /**
+   * @var {file} list
+   * This sets the imported file listed_apps.js to the variable, list.
+  */
   var list = require('../../models/listed_apps.js');
+
+  /**
+   * @var {file} util
+   * This sets the imported file util.js to the variable, util.
+  */
   var util = require('../../../lib/util');
 
   // Create
+  /**
+   * @function put
+   * @param filePath
+   * ATTN: A route or filepath with unqiue data.
+   * @param function(req,res)
+   * This is a function that takes in req and res as parameters
+   * @desc ATTN: What does this do?
+  */
```

Comment:
Routes do not need commented in this manner as they are not actual function definitions.